sympy/simplify/simplify.py

```diff
@@ -251,7 +251,7 @@ def posify(eq):
             eq[i] = e.subs(reps)
         return f(eq), {r: s for s, r in reps.items()}
 
-    reps = {s: Dummy(s.name, positive=True)
+    reps = {s: Dummy(s.name, positive=True, **s.assumptions0)
```

Comment:
You might get conflicting assumptions. Perhaps it would be better to only take those that are not implied by `positive`:
```
>>> Dummy(positive=True).assumptions0
{'positive': True, 'complex': True, 'nonnegative': True, 'real': True, 'imaginar
y': False, 'negative': False, 'nonzero': True, 'nonpositive': False, 'commutativ
e': True, 'zero': False, 'hermitian': True}
>>> _.keys()
dict_keys(['positive', 'complex', 'nonnegative', 'real', 'imaginary', 'negative'
, 'nonzero', 'nonpositive', 'commutative', 'zero', 'hermitian'])
>>> ', '.join(sorted(_))
'commutative, complex, hermitian, imaginary, negative, nonnegative, nonpositive,
 nonzero, positive, real, zero'
```
pos_defined = ['commutative,', 'complex,', 'hermitian,', 'imaginary,', 'negative,', 'nonnegative,', 'nonpositive,', 'nonzero,', 'positive,', 'real,', 'zero']
reps = {s: Dummy(s.name, postive=True, {k:v for k in s.assumptions0 if k not in pos_defined})}
```

Reply:
Hi @smichr I checked all the conflicting assumptions.
```py
>>> Dummy(positive=True).assumptions0
{'positive': True, 'complex': True, 'nonnegative': True, 'real': True, 'imaginar
y': False, 'negative': False, 'nonzero': True, 'nonpositive': False, 'commutativ
e': True, 'zero': False, 'hermitian': True}
```
If all of them are present in this form then there would be no conflict. But if anyone is in negated form of above, then it would imply `positive=False` hence `posify` wouldn't be applied and there would be no conflict.
```py
>>> Dummy(complex=False).is_positive
False
>>> Dummy(hermitian=False).is_positive
False
>>> Dummy(imaginary=True).is_positive
False
>>> Dummy(negative=True).is_positive
False
>>>
```

Reply:
The code is
```python
   reps = {s: Dummy(s.name, positive=True, **s.assumptions0)
                    for s in eq.free_symbols if s.is_positive is None}
```
so only symbols that have uncertain positive-ness are posified.